misk-redis/src/main/kotlin/misk/redis/RedisClusterConfig.kt

```diff
@@ -0,0 +1,21 @@
+package misk.redis
+
+import misk.config.Redact
+import redis.clients.jedis.JedisCluster.DEFAULT_MAX_ATTEMPTS
+import redis.clients.jedis.Protocol
+import wisp.config.Config
+
+class RedisClusterConfig : java.util.LinkedHashMap<String, RedisClusterReplicationGroupConfig>, Config {
```

Comment:
Could you add some docs for what key/value represent? Also could/should this be a data class to match the simpler modelling of config other places?

Reply:
Good point, added more docs particularly in `RedisClusterReplicationGroupConfig` which is the data class. The one here is the top level config which I matched with `RedisConfig` (and DataSourceConfig) and they're not data class due to the constructors.